src/index.html

```diff
@@ -9,10 +9,53 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__blocks">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="stars stars--1">
```